fedora_messaging/message.py

```diff
@@ -434,3 +540,17 @@ def packages(self):
             list(str): A list of affected package names.
         """
         return []
+
+    def dump(self):
+        """
+        Dump message attributes.
+
+        Returns:
+            dict: A dictionary of messge attributes.
+        """
+        return {
```

Comment:
It'd probably be nice to save the message's id as well, so test writers who want to use this functionality can rely on stable message ids.